references/cli/provider.go

```diff
@@ -316,10 +329,8 @@ func getTerraformProviderType(ctx context.Context, k8sClient client.Client, name
 }
 
 func prepareProviderDeleteCommand(c common.Args, ioStreams cmdutil.IOStreams) (*cobra.Command, error) {
-	ctx := context.Background()
-	k8sClient, err := c.GetClient()
-	if err != nil {
-		return nil, err
+	if len(os.Args) < 2 || os.Args[1] != "provider" {
```

Comment:
Any there any possibilities that the second argument is not `provider`?

Reply:
For vela CLI, the args index number is determinate.

It is OK as follows:
```
$ ./vela provider --help

$ bash -c "./vela provider  --help"
```